cmake/Modules/FunctionQbtCXXCompilerAndModeCheck.cmake

```diff
@@ -8,15 +8,15 @@
 
 function(qbt_minimum_cxx_mode_check _min_std)
     # ensure the compiler fully supports the minimum required C++ mode.
-    if(NOT CMAKE_CXX${_min_std}_STANDARD__HAS_FULL_SUPPORT)
+    if (NOT CMAKE_CXX${_min_std}_STANDARD__HAS_FULL_SUPPORT)
```

Comment:
@glassez looks like the situation is worse than expected; with C++17, this won't even work with GCC or Clang either: https://gitlab.kitware.com/search?project_id=541&repository_ref=master&search=CMAKE_CXX17_STANDARD__HAS_FULL_SUPPORT.

Honestly at this point it would just be better to hard-code minimum supported versions (e.g. for C++14 that is GCC >= 5.0, clang >= 5.0, MSVC >= 19.11.25505).